global-cli/index.js

```diff
@@ -166,3 +169,13 @@ function checkNodeVersion() {
     process.exit(1);
   }
 }
+
+function isGitHubBoilerplate(root) {
+  var validFiles = [
+    '.DS_Store', 'Thumbs.db', '.git', '.gitignore', 'README.md', 'LICENSE'
```

Comment:
Can we add 'CNAME' to the the validFiles array? https://help.github.com/articles/adding-or-removing-a-custom-domain-for-your-github-pages-site/